src/lib/core/overlay/overlay.spec.ts

```diff
@@ -295,6 +296,50 @@ describe('Overlay', () => {
     });
 
   });
+
+  describe('scroll strategy', () => {
+    let fakeScrollStrategy: ScrollStrategy;
+    let config: OverlayState;
+
+    beforeEach(() => {
+      config = new OverlayState();
+      fakeScrollStrategy = {
+        attach: jasmine.createSpy('attach spy'),
+        enable: jasmine.createSpy('enable spy'),
+        disable: jasmine.createSpy('disable spy')
```

Comment:
The main reason I avoided adding `isAttached` and `isEnabled` was that I wanted to keep the `ScrollStrategy` interface as lean as possible. Apart from unit tests, I'm not sure that they provide a lot of value.